docusaurus.config.js

```diff
@@ -8,14 +8,21 @@ module.exports = {
   themeConfig: {
     // ...
   },
-  configureWebpack: {
-    module: {
-      rules: [
-        {
-          test: /\.(w|js|ts|tf)$/,
-          use: 'file-loader',
-        },
-      ],
-    },
+  configureWebpack: (config, isServer, utils) => {
+    const { getCacheLoader } = utils;
+    return {
+      module: {
+        rules: [
+          {
+            test: /\.(w|js|ts|tf)$/,
+            use: 'file-loader',
```

Comment:
I believe webpack 5 does not need this anymore, you can use [asset/resource](https://webpack.js.org/guides/asset-modules/)

Reply:
note this section as well https://webpack.js.org/guides/asset-modules/#replacing-inline-loader-syntax